go/test/endtoend/clustertest/main_test.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2019 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clustertest
+
+import (
+	"flag"
+	"net/http"
+	"os"
+	"testing"
+	"vitess.io/vitess/go/test/endtoend/cluster"
+)
+
+var (
+	ClusterInstance *cluster.LocalProcessCluster
```

Comment:
Are any of these variables used outside this package? Or expected to be used that way?
If not, they should start with lowercase.

Reply:
Yes, there are being using in individual testcases e.g. `vtgate_test.go` located in the same package.

Reply:
If it is all in the same package, they can be lowercased. You can leave it for now and revisit at the end.